attributes/default.rb

```diff
@@ -111,9 +113,11 @@
     default['php']['enable_mod']       = '/usr/sbin/phpenmod'
     default['php']['disable_mod']      = '/usr/sbin/phpdismod'
     default['php']['ext_conf_dir']     = '/etc/php/7.0/mods-available'
-  elsif platform?('ubuntu') && node['platform_version'].to_f >= 18.04
-    default['php']['version']          = '7.0.4'
-    default['php']['checksum']         = 'f6cdac2fd37da0ac0bbcee0187d74b3719c2f83973dfe883d5cde81c356fe0a8'
+  elsif platform?('ubuntu') &&
```

Comment:
You can probably just check for 20.04 here and skip the range. 18.04 would already match earlier.